api/server/handlers/environment/enable_pull_request.go

```
@@ -124,9 +160,16 @@ func (c *EnablePullRequestHandler) ServeHTTP(w http.ResponseWriter, r *http.Requ
 			},
 		},
 	)
+	if err != nil {
+		_ = telemetry.Error(ctx, span, err, "error creating workflow dispatch event")
+		c.HandleAPIError(w, r, apierrors.NewErrInternal(err))
```

Any reason not to use the returned error from the line above, as the internal error here?

---

Just wanted to show that I was explicitly swallowing the errors in cases where the HandleAPIError inputs were super interpolated, but I'll add them back in where there is no interpolation.